packages/amplication-client/src/Entity/EntityList.tsx

```diff
@@ -89,6 +91,12 @@ const EntityList: React.FC<Props> = ({ match, innerRoutes }) => {
     };
   }, [refetch, stopPolling, startPolling]);
 
+  const handleEntityClick = () => {
+    trackEvent({
+      eventName: AnalyticsEventNames.UpgradeOnEntityListClick,
```

Comment:
According to the Notion docs you need to add this metadata:
Reason: “Entities limit notification”
Location: App

Reply:
I can't add it. we need to move stigg provider to higher level in order to get the reason for the notification